ArcheologieportaalDialog.js

```diff
@@ -81,6 +85,16 @@ define([
         'rapporten/archeologierapporten/archeologierapporten_opgraving_wv/beheer';
       this.eindverslagen_opgr_wv.href = this.archeologieportaalUrl +
         'rapporten/eindverslagen/eindverslagen_opgraving_wv/beheer';
+      
+      // Processen
+      if (this.authorisationController && this.authorisationController._isBeheerder()
+          && this.procesUrl) {
```

Comment:
die check op this.procesUrl mag hier weg ze, beter dat we zien dat hiet niet ingevuld is, ipv te twijfelen of de rechten niet ok zijn.